ets-demo/etstool.py

```diff
@@ -127,6 +130,21 @@
 
 doc_dependencies = {}
 
+full_app_dependencies = [
+    'eam',
+    'traitsui',
+    'etsdemo',
+    'envisage',
```

Comment:
'envisage' currently has examples/demos contributed to etsdemo, but because of #1428, the examples effectively do not run currently.  
We should either wait until this is resolved to upload the bundles, or not include `envisage` when we first upload the bundles.

Reply:
I removed envisage